### PR TITLE
Use WIF encoding for private keys in Go SDK

### PR DESCRIPTION
**Pull request by aludvik**
This PR depends on PR #670

### CODE REVIEW

**Comment by aludvik** [comment]
Split the commit into more focused commits and added the dependency to vagrant.

File: sdk/go/src/sawtooth_sdk/client/signer.go [hunk 1]
```diff
@@ -91,6 +92,22 @@ func SHA256(input []byte) []byte {
...
 }
 
+// -- WIF --
+
+// PrivToWif converts a private key generated to a WIF string
+func PrivToWif(priv []byte) string {
+	extended := append([]byte{0x80}, priv...)
+	checksum := SHA256(SHA256(extended))[:4]
+	extcheck := append(extended, checksum...)
+	return base58.Encode(extcheck)
```
Comment (dcmiddle) [inline sdk/go/src/sawtooth_sdk/client/signer.go]: Is this just translating to base58? You might need checkencode(). I'm not sure off the top of my head if that's sufficient for wif or if there are other requirments.

**Comment by aludvik** [comment]
We are already using https://github.com/btcsuite/btcd which has the same license.  Additionally, https://github.com/tendermint/go-crypto is using that library but is licensed with Apache 2.0 https://github.com/tendermint/go-crypto/blob/master/LICENSE. I think the license is okay based on that.

**Comment by aludvik** [comment]
Used a private key generated from sawtooth keygen for all the tests.

**Comment by aludvik** [comment]
@vaporos, @TomBarnes found that the ISC license is okay.

**Pull request merged**